.github/workflows/e2e-test.yaml

```diff
@@ -0,0 +1,54 @@
+name: e2e-test
+on:
+  schedule:
+    - cron: '12 3 * * *'
+permissions:
+  id-token: write
+  contents: read
+jobs:
+  run-e2e:
```

Comment:
suggestion (blocking): clarify that this is the AWS e2e test

Reply:
Changed to 'run-e2e-aws-oicd'.